cmd/translate/main_grpc_integration_test.go

```diff
@@ -201,3 +204,179 @@ func Test_DownloadTranslationFile_gRPC(t *testing.T) {
 		})
 	}
 }
+
+// ------------------Service------------------
+
+func randService() *translatev1.Service {
+	return &translatev1.Service{
+		Id:   gofakeit.UUID(),
+		Name: gofakeit.FirstName(),
+	}
+}
+
+func Test_CreateService_gRPC(t *testing.T) {
+	t.Parallel()
+
+	service := randService()
+
+	_, err := client.CreateService(context.Background(), &translatev1.CreateServiceRequest{Service: service})
+	if !assert.NoError(t, err, "Prepare test data") {
+		return
+	}
+
+	actual := status.Code(err)
+	expected := codes.OK
+
+	assert.Equal(t, expected, actual, "want codes.%s got codes.%s", expected, actual)
```

Comment:
Up to you. But to reduce code boilerplate code (tests by nature are very noisy). It makes it easier to read the tests.

```suggestion
	assert.Equal(t, codes.OK, status.Code(err))
```

Reply:
I would agree with you, but when tests fail with 
```go
assert.Equal(t, codes.OK, status.Code(err)) 
```
the expected and actual status codes are not human readable, while 
```go
assert.Equal(t, expected, actual, "want codes.%s got codes.%s", expected, actual)
```
On fail prints human-readable expected and actual status code, so I will stick with latter